blmath/value.py

```diff
@@ -55,7 +55,7 @@ def __float__(self):
     def __int__(self):
         return int(self.value)
     def __round__(self, places):
-        return round(self.value, places)
+        return Value(round(self.value, places), self.units)
```

Comment:
I think this is fixing something I added for Python 3 support, yea? Could you elaborate? Related to the changes in `test_value`?

Reply:
It's an minimize surprise change -- `round(int)` returns an int; `round(float)` returns a float, so `round(Value)` should return a Value. It shouldn't break anything, as you can treat a value as a numeric type and cast to int or float. It's a merge artifact that I thought worth keeping, but if it breaks the api for you, I could go either way.

Reply:
Gotcha. Yea, that makes sense. I think anything using `round` and then doing something else with that value might be expecting a number, but really we should just make Value support whatever that later operation is.